web/client/codechecker_client/cmd/store.py

```diff
@@ -191,6 +193,7 @@ def assemble_zip(inputs, zip_file, client):
     file_to_hash = {}
     file_to_mtime = {}
     missing_source_files = set()
+    suppressed_file_hashes = set()
```

Comment:
this should be `file_hash_with_review_status` or something like this, not just suppress, please add a comment that the content hashes are stored here for source files which contain codechecker review status comments